packages/tools/client-debug-view/src/components/AudienceView.tsx

```diff
@@ -184,10 +209,15 @@ function HistoryView(props: HistoryViewProps): React.ReactElement {
 			styles={{
 				root: {
 					overflowY: "auto",
+                    height: "300px"
 				},
 			}}
 		>
-			<ul>{historyViews}</ul>
+			<div style={{ overflowY: "scroll" }}>
```

Comment:
Do we need this element? Shouldn't the above stack already give us the automatic overflowing?

Reply:
Answer included below.